atomicapp/applogging.py

```diff
@@ -100,6 +101,8 @@ def setup_logging(verbose=None, quiet=None, logtype=None):
         The 'color' level prints out normal log msgs (no cockpit) with color
         """
 
+        # setting the root directory path of code base
+        customOutputFormatter.path = '/'.join(__file__.split('/')[:-1])
```

Comment:
This is where I have set the value of the root directory of the code base, is there a better place to do this? I did it here because this will be the first place logging will start, so!